Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -113,47 +119,39 @@ - (void)setupViews {
-                                              [self.calendarView.trailingAnchor constraintEqualToAnchor:self.trailingAnchor constant:-BPKSpacingBase]
-                                              ]];
-    
+
     self.calendarWeekdayView = [[FSCalendarWeekdayView alloc] initWithFrame:CGRectZero];
-    self.calendarWeekdayView.translatesAutoresizingMaskIntoConstraints = NO;
     self.calendarWeekdayView.calendar = self.calendarView;
     self.calendarWeekdayView.backgroundColor = [BPKColor white];
     [self addSubview:self.calendarWeekdayView];
     
-    UIView *bottomBorder = [[UIView alloc] initWithFrame:CGRectZero];
-    bottomBorder.translatesAutoresizingMaskIntoConstraints = NO;
-    bottomBorder.backgroundColor = [BPKColor gray100];
-    [self.calendarWeekdayView addSubview:bottomBorder];
-    
-    [NSLayoutConstraint activateConstraints:@[
-                                              [self.calendarWeekdayView.topAnchor constraintEqualToAnchor:self.topAnchor],
-                                              [self.calendarWeekdayView.leadingAnchor constraintEqualToAnchor:self.leadingAnchor],
-                                              [self.calendarWeekdayView.trailingAnchor constraintEqualToAnchor:self.trailingAnchor],
-                                              [self.calendarWeekdayView.heightAnchor constraintEqualToConstant:6*BPKSpacingMd]
-                                              ]];
-    
-    [NSLayoutConstraint activateConstraints:@[
-                                              [bottomBorder.bottomAnchor constraintEqualToAnchor:self.calendarWeekdayView.bottomAnchor],
-                                              [bottomBorder.leadingAnchor constraintEqualToAnchor:self.calendarWeekdayView.leadingAnchor],
-                                              [bottomBorder.trailingAnchor constraintEqualToAnchor:self.calendarWeekdayView.trailingAnchor],
-                                              [bottomBorder.heightAnchor constraintEqualToConstant:1.0]
-                                              ]];
+    self.bottomBorder = [[UIView alloc] initWithFrame:CGRectZero];
+    self.bottomBorder.backgroundColor = [BPKColor gray100];
+    [self.calendarWeekdayView addSubview:self.bottomBorder];
     
     self.yearPill = [[BPKCalendarYearPill alloc] initWithFrame:CGRectZero];
     self.yearPill.hidden = YES;
     [self addSubview:self.yearPill];
-    
-    [NSLayoutConstraint activateConstraints:@[
-                                              [self.yearPill.topAnchor constraintEqualToAnchor:self.calendarWeekdayView.bottomAnchor constant:BPKSpacingLg],
-                                              [self.yearPill.centerXAnchor constraintEqualToAnchor:self.centerXAnchor]
-                                              ]];
+}
+
+- (void)layoutSubviews {
+    [super layoutSubviews];
+    // NOTE: This class uses manual layout rather than Auto Layout because it needs to be usable as a Native UI
```

Comment:
```suggestion
    // NOTE: This class uses manual layout rather than Auto Layout because it needs to be usable as a React Native UI
```

Reply:
I like what we already have because it makes it clear not to convert the file to auto layout